src/Compilers/CSharp/Portable/Lowering/AsyncRewriter/AsyncRewriter.AsyncIteratorRewriter.cs

```diff
@@ -15,6 +15,8 @@ internal partial class AsyncRewriter : StateMachineRewriter
         /// </summary>
         private sealed class AsyncIteratorRewriter : AsyncRewriter
         {
+            private readonly TypeSymbol _elementType;
```

Comment:
>_elementType [](start = 40, length = 12)

If this value is only used in one location, consider using `stateMachineType.ElementType` directly there rather than creating a field.